chaossearch/client/operation_create_object_group.go

```diff
@@ -71,6 +71,11 @@ func marshalCreateObjectGroupRequest(req *CreateObjectGroupRequest) ([]byte, err
 		body["partitionBy"] = req.PartitionBy
 	}
 
+	if req.Format == "JSON" {
+		var format = body["format"].(map[string]interface{})
+		format["pattern"] = req.Pattern
```

Comment:
Shouldn’t we set the pattern when the type is log?

Reply:
You're right, fixed